### PR TITLE
A significant change in apoc.periodic.iterate in APOC 4.0

### PR DESCRIPTION
**Pull request by viveksaran71**

### CODE REVIEW

**Review by sarmbruster — APPROVED** [review]
LGFM - thanks for contributing this.

**Review by sarmbruster — APPROVED** [review]
LGFM

File: articles/a-significant-change-in-apoc.periodic.iterate-in-apoc-4.0.adoc [hunk 1]
```diff
@@ -0,0 +1,47 @@
+= A significant change in apoc.periodic.iterate() in apoc 4.0
+// CHANGE THIS, REMOVE COMMENTS BEFORE MERGE
+// ANY SLUG CHANGES WILL CAUSE POSTS TO BE RE-CREATED, BREAKING EXISTING LINKS
```
Comment (Mogztter) [inline articles/a-significant-change-in-apoc.periodic.iterate-in-apoc-4.0.adoc:3]:
```suggestion
```

File: articles/a-significant-change-in-apoc.periodic.iterate-in-apoc-4.0.adoc [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+:slug: a-significant-change-in-apoc-periodic-iterate-in-apoc-4-0
+:author: Vivek Saran
+// relevant versions
+:neo4j-versions: 4.0, 4.1
+// see taxonomy in readme, remove this comment
```
Comment (Mogztter) [inline articles/a-significant-change-in-apoc.periodic.iterate-in-apoc-4.0.adoc:8]:
```suggestion
```

**Review by Mogztter — APPROVED** [review]
:+1:

**Review by danielterlizzi — APPROVED** [review]
Reviewed and approved by Stefan Armbruster

**Pull request merged**